### PR TITLE
Frontend Pipeline Fixes + Sonar

### PR DESCRIPTION
**Pull request by mauantoniojw**

### CODE REVIEW

**Review by viannbreslerAccso — APPROVED** [review]

File: .github/workflows/veracode_frontend.yml
Comment (hgo-ds) [inline .github/workflows/veracode_frontend.yml:1]: **Fix:**
Please revert these changes, since we will not use varacode for cofinit-x.

File: .gitignore
Comment (hgo-ds) [inline .gitignore:1]: **Fix:**

Move the ignore-entries into a .gitignore file as close as possible to the ignored files. Do not edit a global .gitignore file.
